README.md

```diff
@@ -171,3 +171,26 @@ egg.
 You can use the `GITHUB_TOKEN` environment variable to specify the token
 
     $ GITHUB_TOKEN=xxx python -m tools.copy_maintainers
+
+## Integration with `pre-commit`
+
+In any addons repo, you can use these pre-commit hooks:
+
+```yaml
+# .pre-commit-config.yaml file
+repos:
+  - repo: https://github.com/OCA/maintainer-tools
+    rev: master # Better use a tag/commit!
+    hooks:
+      # Use each script's `--help` to understand the args
+      - id: oca-gen-addon-readme
+        args:
+          - --addons-dir=.
+          - --org-name=OCA
+          - --repo-name=server-tools
+          - --branch=13.0
+      - id: oca-gen-addons-table
```

Comment:
remove this

Reply:
This is just an example that tells you what you *can* do.

Instead of removing it, I added `stages: [manual]`, so this trigger doesn't run automatically. I also added a comment explaining that this could produce conflicts easily and should be used with care.

These are docs anyway, it's not like the feature is buggy, it's just that users need to know what it does and the problems it has.